include/nbl/builtin/hlsl/type_traits.hlsl

```diff
@@ -13,7 +132,7 @@
 namespace nbl::hlsl::type_traits
 {
 
-#ifdef __HLSL_VERSION // HLSL
+#if 1 //def __HLSL_VERSION // HLSL
```

Comment:
wait waht?

Reply:
That was to allow intellisense color the if block. I forgot to undo it before committing.